### PR TITLE
Bosnia and Herzegovina holidays update

### PR DESCRIPTION
**Pull request by KJhellico**

### CODE REVIEW

**Comment by KJhellico** [comment]
Tests needs to be rewritten with `assertHolidaysName`, now I see it. It will be done ASAP.

File: test/countries/test_bosnia_and_herzegovina.py [hunk 1]
```diff
@@ -10,285 +10,281 @@
...
-class TestBA(unittest.TestCase):
+class TestBA(TestCase):
     def setUp(self):
-        self.holidays = holidays.BA(observed=False)
-        self.holidays_observed = holidays.BA(observed=True)
-
-    def test_new_years(self):
-        for dt in (date(2010, 12, 31), date(2017, 1, 3)):
-            self.assertNotIn(dt, self.holidays)
+        self.holidays = BosniaAndHerzegovina(observed=False)
```
Comment (arkid15r) [inline test/countries/test_bosnia_and_herzegovina.py:20]: Not sure why we're testing non-default (observed=False) behaviour here.

**Review by arkid15r — COMMENTED** [review]
LGTM

File: test/countries/test_bosnia_and_herzegovina.py (hunk 1 above)
Comment (KJhellico) [reply test/countries/test_bosnia_and_herzegovina.py:20]: Before it was like that (`self.holidays = holidays.BA(observed=False) …`), so I left it. :)

**Comment by arkid15r** [comment]
Merged! Thanks for reworking this, @KJhellico!

**Pull request merged**